monofony/core-bundle/0.1/manifest.json

```
@@ -0,0 +1,21 @@
+{
+    "bundles": {
+        "Monofony\\Bundle\\CoreBundle\\MonofonyCoreBundle": ["all"]
+    },
+    "container": {
+        "locale": "fr_FR",
+        "email_contact": "contact@app_name.com",
+        "email_name": "Contact AppName",
+        "email_sender": "no-reply@app_name.com"
```

I'd suggest to use another domain, to avoid having issues if one day the `app_name.com` exists 😛 
(by the way, the `app_name.com` domain name is invalid because domain names can contain only alphanumeric and hyphen characters)
```suggestion
        "email_contact": "contact@example.com",
        "email_name": "Contact AppName",
        "email_sender": "no-reply@example.com"
```